html/pfappserver/root/static.alt/src/App.vue

```diff
@@ -39,6 +39,19 @@
             <b-dropdown-divider></b-dropdown-divider>
             <b-dropdown-item to="/logout">{{ $t('Log out') }}</b-dropdown-item>
           </b-nav-item-dropdown>
+          <b-nav-item-dropdown right v-if="tenant && tenant.id === 0">
+            <template v-slot:button-content>
+              <icon name="layer-group"></icon> {{ tenant_mask_name }}
+            </template>
+            <b-dropdown-item-button v-for="tenant in tenants" :key="tenant.id"
```

Comment:
I propose to add a `<b-dropdown-item-header>` element at the top of the list that would says *Tenants*.